src/pages/settings/Security/TwoFactorAuth/CodesPage.js

```diff
@@ -64,65 +67,67 @@ function CodesPage(props) {
-                        onPress={() => Navigation.navigate(ROUTES.SETTINGS_2FA_VERIFY)}
-                        isDisabled={isNextButtonDisabled}
-                    />
-                </FixedFooter>
+                            ) : (
+                                <>
+                                    <View style={styles.twoFactorAuthCodesContainer}>
+                                        {Boolean(props.account.recoveryCodes) &&
+                                            _.map(props.account.recoveryCodes.split(', '), (code) => (
+                                                <Text
+                                                    style={styles.twoFactorAuthCode}
+                                                    key={code}
+                                                >
+                                                    {code}
+                                                </Text>
+                                            ))}
+                                    </View>
+                                    <View style={styles.twoFactorAuthCodesButtonsContainer}>
+                                        <Button
+                                            text={props.translate('twoFactorAuth.copyCodes')}
+                                            medium
+                                            onPress={() => {
+                                                Clipboard.setString(props.account.recoveryCodes);
+                                                setIsNextButtonDisabled(false);
+                                            }}
+                                            style={styles.twoFactorAuthCodesButton}
+                                        />
+                                        <Button
+                                            text={props.translate('common.download')}
+                                            medium
+                                            onPress={() => {
+                                                localFileDownload('two-factor-auth-codes', props.account.recoveryCodes);
+                                                setIsNextButtonDisabled(false);
+                                            }}
+                                            style={styles.twoFactorAuthCodesButton}
+                                        />
+                                    </View>
+                                </>
+                            )}
+                        </View>
+                    </Section>
+                    <FixedFooter style={[styles.twoFactorAuthFooter]}>
+                        <Button
+                            success
+                            text={props.translate('common.next')}
+                            onPress={() => Navigation.navigate(ROUTES.SETTINGS_2FA_VERIFY)}
+                            isDisabled={isNextButtonDisabled}
+                        />
+                    </FixedFooter>
```

Comment:
Seems we accidentally included Fixed inside scroll containers and that led to this regression https://github.com/Expensify/App/issues/20138